core_domain.process_model_execution/src/flow_node_handler/activity_handler/call_activity_handler.ts

```diff
@@ -34,6 +36,9 @@ export class CallActivityHandler extends ActivityHandler<Model.Activities.CallAc
   private processDefinitionRepository: IProcessDefinitionRepository;
   private processInstanceRepository: IProcessInstanceRepository;
   private resumeProcessService: IResumeProcessService;
+  private processInstanceManagementService: IProcessInstanceManagementService;
+
+  private processInstanceId: string;
```

Comment:
Ich hab mich erst gefragt, warum hier die ID der Prozessinstanz gespeichert wird, bis ich dann im weiteren Verlauf festgestellt hab, dass das hier die ID des Subprozesses ist.

Bitte etwas sorgfältiger mit den Begriffen umgehen, der Unterschied zwischen `ProcessInstanceId` und `SubprocessInstanceId` ist grade in dem Handler hier kriegsentscheidend.

```suggestion
  private subProcessInstanceId: string;
```